inference/core/workflows/execution_engine/compiler/syntactic_parser.py

```diff
@@ -51,9 +52,15 @@ def build_workflow_definition_entity(
     block_manifest_types_union = Union[steps_manifests]
     block_type = Annotated[block_manifest_types_union, Field(discriminator="type")]
     return create_model(
-        "WorkflowSpecificationV1",
+        "WorkflowSpecificationV2",
```

Comment:
Is that change needed

Reply:
initially added to make sure it was fetching the new version on the frontend, will revert back